public/scripts/app.js

```diff
@@ -0,0 +1,26 @@
+angular
+  .module('bookly', ['ngRoute'])
+  .config(config)
+
+config.$inject = ['$routeProvider', '$locationProvider'];
+function config(   $routeProvider,  $locationProvider   ) {
+  $routeProvider
+    .when('/', {
+      templateUrl: 'templates/books-index.html',
+      controller: 'BooksIndexController',
+      controllerAs: 'bic'
+    })
+    .when('/books/:id', {
+      templateUrl: '../templates/books-show.html',
```

Comment:
This should just start with /templates, rather than being a relative path from where the frontend will be at that point.